app/src/config/env/local.py

```diff
@@ -0,0 +1,16 @@
+#
+# Configuration for local development environments.
+#
+# This file only contains overrides (differences) from the defaults in default.py.
+#
+
+import pydantic.types
+
+from .. import default
+
+config = default.default_config()
+
+config.database.password = pydantic.types.SecretStr("secret123")
+config.database.hide_sql_parameter_logs = False
+config.logging.format = "human_readable"
+config.logging.enable_audit = False
```

Comment:
Would there be a way to have this check for a file named `local_overrides.py` and use that? I'm thinking about local testing where I want to tinker with environment variables, but not check them in - especially when working with passwords or keys to access services. We could add this `local_overrides.py` file to the `.gitignore`.

Adding something like this to the end of this file:

```py
if os.path.exists("local_overrides.py"):
     importlib.import_module("local_overrides") # whatever the right path/params are
```